{"name": "test_case", "status": "broken", "statusDetails": {"message": "NameError: name 'data' is not defined", "trace": "cmdoption = ('test', dict_values(['102.3.154.1', 4444]))\n\n    def test_case(cmdoption):\n        print(\"测试环境验证\")\n        env, datas = cmdoption\n>       print(f\"环境 ： {env} , 数据：{data}\")\nE       NameError: name 'data' is not defined\n\ntest_env.py:5: NameError"}, "start": 1594622108131, "stop": 1594622108131, "uuid": "f9a27d03-f930-4c32-9c0c-05f5840057cc", "historyId": "69d176044c92130a2d804562133548ba", "testCaseId": "f503f3c2e58a20cc58b09b61985b8f83", "fullName": "testcases.test_env#test_case", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_env"}, {"name": "host", "value": "bogon"}, {"name": "thread", "value": "58895-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_env"}]}